api/src/main/resources/messages.properties

```diff
@@ -182,6 +182,9 @@ coreapps.noActiveVisit.description=This patient is not in an active visit. If th
 
 coreapps.deadPatient = The patient is deceased - {0} - {1}
 coreapps.deadPatient.description=This patient is deceased. If you wish to view or edit details of a previous visit, choose a visit from the list on the left.
+coreapps.markPatientDead.label=Mark Patient Dead
+coreapps.markpatientdeceased.dateofdeath=Date of death
```

Comment:
As for the message key, wouldn't it be more readable to capitalise each word's first character? e.g coreapps.markPatientDeceased.dateOfDeath